app/models/computer_game.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class ComputerGame < ApplicationRecord
+  include ActiveModel::Validations
+
+  belongs_to :user
+  has_one :computer, dependent: :destroy
+  has_many :games, dependent: :destroy
+
+  validates_with ComputerGameValidator
```

Comment:
this is in general a good idea! nice! but imo you can use here simple presence validators :) and do a separate validation for check if user is a game owner